### DISCUSSION TITLE
Filecoin on Token Terminal

### DISCUSSION THREAD

**Discussion by rbval** [filecoin-project/community#100] [Project Showcase]
Hi there,

we at Token Terminal just launched the first version of the Filecoin dashboard: https://terminal.tokenterminal.com/dashboard/Filecoin. Currently it only has miner tips calculated towards revenue. We’re trying to find a good way to get the base fees in a historical way in the near future. Our initial approximation is that base fees are 5% of total fees. We are excluding miner rewards from these calculations.

Here’s also a nice screenshot from our main site (https://www.tokenterminal.com):
![image](https://user-images.githubusercontent.com/6399810/110641015-29326200-81ba-11eb-8450-30bfeb21718e.png)

**Comment by jennijuju** [comment]
@rbval  this is cool! Thank you for sharing!
